src/Illuminate/Database/Schema/Builder.php

```diff
@@ -200,6 +200,35 @@ public function create($table, Closure $callback)
         }));
     }
 
+    /**
+     * Create a new pivot table for a many-to-many relation on the schema.
+     *
+     * @param \Illuminate\Database\Eloquent\Model|string $firstModel
+     * @param \Illuminate\Database\Eloquent\Model|string $secondModel
+     * @param Closure|null $callback
```

Comment:
Please make sure that Docblocks match the styling of the rest of the framework and use FQN for classes.

Reply:
Fixed it.

Reply:
There should also be two spaces between the type and param name.

Reply:
There is two spaces between the type and param name in [631ad33](https://github.com/laravel/framework/pull/35397/commits/631ad332ae3c4b92ae0f4a287b4681f3078796d7)

Reply:
There's still some StyleCI failures.

Reply:
Fixed it.